src/js/components/Main/Home/Home.container.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState, useEffect} from 'react';
+import Home from './Home.jsx'
+
+
+function HomeContainer(props) {
+
+  const [inputText, setInputText] = useState("");
+  const [concerts, setConcerts] = useState([]);
+  const [helpList, setHelpList] = useState([]);
+
+  useEffect(() => {
+    props.getConcerts(getConcerts)
+  });
+
+  function getConcerts(concerts) {
```

Comment:
Same as in other PR: use `useCallback`. In this case you can even pass `setConcerts` directly without need in `getConcerts` function